### PR TITLE
Add google login to Login.js

### PR DESCRIPTION
**Pull request by aberonja**
Installed react-google-login package
Added GoogleLogin component to Login.js file and function for console loging result for testing

### CODE REVIEW

File: web/src/main/frontend/src/pages/Login.js [hunk 1]
```diff
@@ -1,8 +1,12 @@
 import React from 'react';
 import {Link} from 'react-router-dom';
 import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import GoogleLogin from 'react-google-login';
 
 class Login extends React.Component {
+  responseGoogle = response => {
+    console.log(response);
+  };
```
Comment (iscekic) [inline web/src/main/frontend/src/pages/Login.js]: nit: add empty line below this line.

File: web/src/main/frontend/src/pages/Login.js [hunk 2]
```diff
@@ -18,9 +22,13 @@ class Login extends React.Component {
         <Link className="button__link" to="/login/guest">
           I want to be an Execom employee
         </Link>
-        <Link className="button__link" to="/">
-          I am an Execom employee
-        </Link>
+        <GoogleLogin
+          clientId="770480684829-sn3uug83qp5mmo96906ju2svj2pbt0sq.apps.googleusercontent.com"
```
Comment (iscekic) [inline web/src/main/frontend/src/pages/Login.js]: Move this string to a separate file, eg. `env.js`. Inside that file, add a comment explaining what the string is and where it is from.

File: web/src/main/frontend/src/pages/Login.js (hunk 1 above)
Comment (aberonja) [reply web/src/main/frontend/src/pages/Login.js]: Should I always have an empty line between functions/methods and render?

Comment (iscekic) [reply web/src/main/frontend/src/pages/Login.js]: Yes, always have an empty line between methods.

File: web/src/main/frontend/src/base/env.js [hunk 3]
```diff
@@ -0,0 +1,11 @@
+// Any application that uses OAuth 2.0 to access Google APIs must have
+// authorization credentials that identify the application to Google's OAuth 2.0 server.
+
+// Visit the Google API Console on this link https://console.developers.google.com/
+// to obtain OAuth 2.0 credentials such as a client ID and client secret
+// that are known to both Google and your application. The set of values varies based on what type of application you are building.
+// For example, a JavaScript application does not require a secret, but a web server application does.
+
+// This string is client ID for Google oAuth 2.0
+export const givenClientId =
```
Comment (iscekic) [inline web/src/main/frontend/src/base/env.js]: A better variable name would be `googleClientId`.

**Pull request merged**